server/app/src/main/resources/indexer.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
Sorry, but here, files are template to help the user but not real configuration files.

Real configuration files should be declared in dockerfiles/run/spring